### ISSUE TITLE
Only working using Leaflet 0.7.7 ?

### ISSUE THREAD

**Issue by skol-pro** [Leaflet/Leaflet.draw#516]
Hi.

After some tests, here are the issues I get using different Leaflet versions:
- **Leaflet 1.0 beta 2**: `'_leaflet_id' of undefined` when clicking on edit button.
- **Leaflet 1.0-dev**: Ending the draw of a circle make it small.

So, is Leaflet 0.7.7 the only supported version ?

**Comment by ddproxy** [comment]
This is right, we have not fully moved to Leaflet 1.0 yet, there is a https://github.com/Leaflet/Leaflet.draw/tree/leaflet-master branch for 1.0 development (but I haven't really touched it lately)

**Comment by skol-pro** [comment]
Actually I managed to get something that works using L.Draw 0.2.4-dev and Leaflet 0.7.2.

**Comment by chugcup** [comment]
See https://github.com/Leaflet/Leaflet.draw/pull/596
